### PR TITLE
Messaging doc release note changes for 3.1

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

### CODE REVIEW

File: install/jms/docs/JMS3.1-ReleaseNotes.html [hunk 1]
```diff
@@ -67,22 +65,20 @@ <h2>Jakarta EE Messaging TCK Facts</h2>
...
-      Edition 8 (1.8) and 11.</p>
+    <p>The Jakarta EE Messaging TCK tests have been built with JDK 11 and tested
+      with Java<font size="-2"><sup>TM</sup></font> Platform, Standard Edition
+      11 and 17.</p>
     <p>The Jakarta EE Messaging TCK tests have been run on the following
       platforms:</p>
     <ul>
-      <li>CentOS Linux 7</li>
-      <li>Alpine Linux v3.12</li>
+      <li>TBD: include OS name + version</li>
```
Comment (gurunrao) [inline install/jms/docs/JMS3.1-ReleaseNotes.html:74]: Shall we continue with CentOS Linux 7 and Alphine Linux V3.12?

Comment (scottmarlow) [reply install/jms/docs/JMS3.1-ReleaseNotes.html:74]: I'm not sure of which operation systems will be used by implementations to test with.  As a start, we can check which operating system is used for the [Eclipse CI](https://ci.eclipse.org/jakartaee-tck/job/checkOS/) which appears to currently be using `Debian GNU/Linux 10`.

Comment (gurunrao) [reply install/jms/docs/JMS3.1-ReleaseNotes.html:74]: +1  for "Debian Linux 10" as in https://ci.eclipse.org/jakartaee-tck/job/checkOS/7/console

Comment (scottmarlow) [reply install/jms/docs/JMS3.1-ReleaseNotes.html]: Updated to `Debian Linux 10`

**Comment by scottmarlow** [comment]
Requested changes have been made.

**Review by gurunrao — APPROVED** [review]

**Pull request merged**